tests/spread/integration/gearman-job-server/task.yaml

```diff
@@ -0,0 +1,7 @@
+summary: Integration tests for gearman-job-server
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
+  rootfs="$(install-slices gearman-job-server_libs)"
+  chroot "${rootfs}/" gearmand --help
```

Comment:
Let's change this to executing `gearmand -h`.

Also, let's follow the convention in other integration tests and insert a blank line between the `rootfs` & `chroot` lines, like this:
```
  rootfs="$(install-slices gearman-job-server_libs)"

  chroot "${rootfs}/" gearmand -h
```